recipe/meta.yaml

```diff
@@ -1,46 +1,90 @@
 {% set name = "splink" %}
-{% set version = "3.6.0" %}
+{% set version = "3.9.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6bbef574b12ebf74e112960e2bc5d8baf3ca60f5d889a581dafb2f29173e90a8
+  sha256: 891996d3eaa6ff58cc3e3cd97c0aa0c45246a8b1c858cd4fe77f617e190f20ee
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38 or s390x]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
+# the requirements refer to the duckdb-less installation documented at
+# https://moj-analytical-services.github.io/splink/getting_started.html#duckdb-less-installation
 requirements:
   host:
-    - python >=3.7,<4.0
+    - python
     - pip
-    - poetry >=0.12
+    - setuptools
+    - wheel
+    - poetry
+    - altair >=5.0.1
```

Comment:
Overall, the whole list of https://github.com/moj-analytical-services/splink/blob/v3.9.10/scripts/duckdbless_requirements.txt should go to the run section instead of the host section. These dependencies server no purpose at build time.

Also, these dependencies could go to an extra output.
Now if the user environment provides duckdb outside of conda, these may not even be needed.
So it could be three outputs:
- splink-core
- splink-duckdb-less - depends on splkink-core + duckdbless_requirements.txt
- splink (which would be commented because of missing python-duckdb).

Reply:
Also, we are missing sqlalchemy >=1.4.49,<2 on main.

Reply:
done.